shcomplete/seq2seq_prediction.ini

```diff
@@ -0,0 +1,17 @@
+[ints]
+# length of the sequence we want to predict the next command
+seq_len = 9
```

Comment:
Not final value. I need to re-generate the corpus because the min size of each history has been fixed to 10 commands.
Whereas seq_len = 50 or even 100 might be better